Scripts/Modules/WorkspaceModule.cs

```diff
@@ -149,10 +164,20 @@ internal void CreateWorkspace(Type t, Action<IWorkspace> createdCallback = null)
 				workspaceCreated(workspace);
 		}
 
+		internal void ProcessInputInWorkspaces(ConsumeControlDelegate consumeControl)
```

Comment:
How about simply ProcessInput? Since it is in the WorkspaceModule, seems redundant that we would specify InWorkspaces, no?